board.py

```diff
@@ -0,0 +1,28 @@
+import tkinter as tk
+from piece import *
+from tile import *
+from Appp import *
+    
+        
+class Game():
+    def __init__(self):
+        self.players = ['red','yellow']
+#         self.num_pieces = {color:12 for color in self.players}
+        self.turn = 0
+        self.selected = (None, None)
+        self.Board = []
+        for row_idx in range(8):
+            row = []
+            for col_idx in range(8):
+                newTile = Tile(row_idx, col_idx, tile_colors[(row_idx + col_idx) % 2])
+                if newTile.color == tile_colors[1]:
+                    if row_idx in range(0,3):
+                        newPiece = Piece("yellow", row_idx, col_idx)
+                        newTile.addPiece(newPiece)
+                    if row_idx in range(5,8):
+                        newPiece = Piece("red", row_idx, col_idx)
+                        newTile.addPiece(newPiece)
```

Comment:
You can extract the repeated line `newTile.addPiece(newPiece)` out of the conditions since the newTile is added in every case